flink-python/pyflink/table/sinks.py

```diff
@@ -49,24 +49,20 @@ class CsvTableSink(TableSink):
     :param write_mode: The write mode to specify whether existing files are overwritten or not.
     """
 
-    def __init__(self, field_names, field_types, path, field_delimiter=',', num_files=1,
+    def __init__(self, field_names, field_types, path, field_delimiter=',', num_files=-1,
```

Comment:
@dianfu could you have a look at the pyflink changes?

Reply:
The pyflink related changes LGTM.